### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#76]
We *need* a proper documentation.

**Comment by kmike** [comment]
//cc @lopuhin what do you think about new docs (http://eli5.readthedocs.io/en/latest/)? Do you have suggestions about how to improve them, or should we close this ticket as what we have is enough for 0.1 release?

**Comment by lopuhin** [comment]
@kmike the docs look great, especially the tutorials! 👍 
I think they are totally fine for the release.

**Comment by kmike** [comment]
Ok! I'm closing it then. We can open new tickets for specific documentation problems.
